tests/integration/it_reservations_test.py

```diff
@@ -7,35 +7,29 @@
 from timeless.restaurants.models import Reservation
 
 
-@pytest.mark.skip("reservations.list does not implemented yet")
 def test_list(client):
     reservation = factories.ReservationFactory()
     response = client.get(url_for("reservations.list"))
     assert response.status_code == HTTPStatus.OK
-    assert reservation.comment in response.data
+    assert reservation.comment in response.data.decode('utf-8')
 
 
 def test_create(client):
-    reservation = factories.ReservationFactory.build()
-    # convert reservation instance to dict
-    reservation_dict = {
-        column.name: str(getattr(reservation, column.name))
-        for column in reservation.__table__.columns
-    }
-    url = url_for("reservations.create_reservation")
-    response = client.post(url, data=reservation_dict)
-    assert response.status_code == HTTPStatus.CREATED
+    reservation = factories.ReservationFactory.get_dict()
+    url = url_for("reservations.create")
+    response = client.post(url, data=reservation)
+    assert response.status_code == HTTPStatus.FOUND
```

Comment:
@vladarefiev in the case of creating an entity, the server should return ``201 CREATED``, not ``FOUND``. Can you change that or is it too much work?

Reply:
@amihaiemil no, you are wrong. Here is a templates, no `201` is here. particularly this view are redirecting you if success

Reply:
@vladarefiev I see, ok